mediabridge-frontend/src/components/MovieItem.tsx

```diff
@@ -14,23 +28,39 @@ const MovieItem = ({
   movie: Movie;
   onRemove: () => void;
 }) => {
+  const [liked, setLiked] = useState<"up" | "down" | null>(null);
+  const pastelIndex = movie.year % pastelBackgrounds.length;
+  const pastelBg = pastelBackgrounds[pastelIndex];
```

Comment:
I'm not a React expert, but I think you need to `useRef()` for things like this, or else it will randomly change colors any time the `MovieItem` is re-rendered.